### PR TITLE
Add command for changing the e-mail for a team.

### PR DESCRIPTION
**Pull request by jziglar**

### CODE REVIEW

File: standup.py [hunk 1]
```diff
@@ -98,6 +98,29 @@ def standup_teams_remove(self, msg, name):
             teams.remove(team)
         return 'Team removed.'
 
+    @arg_botcmd('email', type=str)
+    @arg_botcmd('name', type=str, default=None)
+    def standup_team_configure(self, msg, email, name):
```
Comment (gbin-argo) [inline standup.py:103]: shouldn't it be name then email?

File: standup.py [hunk 2]
```diff
@@ -98,6 +98,29 @@ def standup_teams_remove(self, msg, name):
...
+            for team in teams:
+                if name:
+                    if team.name == name:
+                        break
+                    elif msg.room.frm == team.room:
+                        break
+            else:
+                if name:
+                    return f'Cannot find the team {name}'
+                else:
```
Comment (gbin-argo) [inline standup.py]: no need to else :)

**Review by gbin-argo — CHANGES_REQUESTED** [review]

File: standup.py (hunk 1 above)
Comment (jziglar) [reply standup.py:103]: If the order is switched, how would you specify just the e-mail and deduce the team name from the room?

File: standup.py (hunk 2 above)
Comment (jziglar) [reply standup.py]: If we didn't define name, and attempted to deduce the team from the room name, wouldn't this else return the error message regarding the failure being due to the room, and not the team name?

File: standup.py (hunk 1 above)
Comment (gbin-argo) [reply standup.py:103]: hmm ok good point

**Review by gbin-argo — APPROVED** [review]

**Pull request merged**